### PR TITLE
Upgrade sandpack to fix code cutting issue in view only editor

### PR DESCRIPTION
**Pull request by harish-sethuraman**
Addresses 
- https://github.com/reactjs/reactjs.org/issues/3308#issuecomment-1046023982

### CODE REVIEW

**Comment by harish-sethuraman** [comment]
I've upgraded `0.13.16-experimental.0` to `0.14.1` and build fails @danilowoz Is there any recommended version to upgrade?

File: beta/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "check-all": "npm-run-all prettier lint:fix tsc"
   },
   "dependencies": {
-    "@codesandbox/sandpack-react": "0.13.16-experimental.0",
+    "@codesandbox/sandpack-react": "0.14.1",
```
Comment (danilowoz) [inline beta/package.json:25]: React docs use a different Sanpack release channel (`experimental`), as this application has some hard constraints on peer-dependency. To be clearer, React docs use a specific React version (https://github.com/reactjs/reactjs.org/blob/main/beta/package.json#L38) that we still can't release for most of the Sandpack users. 

I'm still waiting to address one more fix on Sandpack and then I'll release a new `experimental` that will fit here.

Comment (danilowoz) [reply beta/package.json:25]: Hey @harish-sethuraman, feel free to try `v0.14.3-experimental.0`.
I would suggest following this checklist: https://github.com/reactjs/reactjs.org/pull/4337#issue-1138571647

Comment (harish-sethuraman) [reply beta/package.json]: Verified the checklists and works fine

**Comment by gaearon** [comment]
seems worse after the fix because now it's focusable and has a cursor in it. also, you can "tab" inside.

**Pull request merged**